### PR TITLE
Bump to Alpine version

### PR DESCRIPTION
**Pull request by macedogm**
Bump to Alpine version to fix some CVEs in the base image.

### CODE REVIEW

**Review by jiaqiluo — APPROVED** [review]
LGTM

You may need to add a second reviewer, and cut a tag once the PR is merged.

**Review by jakefhyde — APPROVED** [review]

**Comment by macedogm** [comment]
@jiaqiluo thanks for the info. Since this is the first time that I'm doing this kind of version bump, after I create the new tag, could you point me to where in Rancher (or the charts?) do I need to update to reflect the new tag, please?

**Comment by jiaqiluo** [comment]
@macedogm  AFAIK, rancher/fleet is the only place where the image rancher/tekton-utils is used: https://github.com/rancher/fleet/blob/master/charts/fleet/charts/gitjob/values.yaml#L5-L7

You can search in the rancher/fleet repo for relevant commits/PRs

**Pull request merged**
